bin/citgm.js

```diff
@@ -70,40 +75,42 @@ if (!citgm.windows) {
 
 app.parse(process.argv);
 
+var config = app.config ? util._extend(require(resolve(process.cwd(), app.config)), app) : app;
```

Comment:
As in you think this is a typo? No, this isn't `require.resolve`. I'm pulling in `resolve-from` to do effectively a `require.resolve` from `process.cwd()`, and then requiring that.

Open to cleaner approaches though.